Jenkinsfile

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent none
+  environment {
+    BASE_DIR="src/github.com/elastic/hey-apm"
+    APM_SERVER_BASE_DIR = "src/github.com/elastic/apm-server"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+    GO_VERSION = "${params.GO_VERSION}"
+    APM_SERVER_VERSION = "${params.APM_SERVER_VERSION}"
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS') 
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  parameters {
+    string(name: 'GO_VERSION', defaultValue: "1.10.3", description: "Go version to use.")
```

Comment:
this is fine, but we can also jump to 1.11.2

Reply:
done